### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by PuneetP16**
Added: Avatar and Badge components and style documentation pages.

### CODE REVIEW

**Comment by m26unkwn** [comment]
UI is great! especially the color combination 

Things I think should be different.

1.  In code highlight you used some time comments without the HTML comment syntax.

image

2. I would suggest that for every badge variant create a separate section. 
eg: for text, variant create one section and for button separate section similarly for icons.

**Comment by PuneetP16** [comment]
> UI is great! especially the color combination
> 
> Things I think should be different.
> 
> 1. In code highlight you used some time comments without the HTML comment syntax.
> 
> image
> 
> 2. I would suggest that for every badge variant create a separate section.
>    eg: for text, variant create one section and for button separate section similarly for icons.

Thank you for your compliment and review. And the suggestions you mentioned, I am incorporating it as per my UI.

**Comment by rahulyadav139** [comment]
The class naming convention is super easy and understandable.  The file looks neat and clean. I think you have created your own customized typography. Awesome.

**Comment by PuneetP16** [comment]
> The class naming convention is super easy and understandable. The file looks neat and clean. I think you have created your own customized typography. Awesome.

Thanks for the awesome feedback.✌️

**Comment by deekshithmd** [comment]
looks good for me and the responsiveness of the page is good one.

**Comment by muskanbagrecha** [comment]
1. Improve padding in badges
2. Align containers and text
3. Increase font-weight of badges
4. Change the color of badges for a better visual appeal.

**Comment by manojsarna** [comment]
1. improve badges in every line.

2. improve color combinations in the website.

3.  improve contrast throughout the website.

4. improve badge page.

5. Use one color for the backgrounds. ( can be a shade of the primary eg either dark or light) its up to you.

**Review by manojsarna — APPROVED** [review]
everything looks good , but please check following pointers :)

improve badges in every line.

improve color combinations in the website.

improve contrast throughout the website.

improve badge page.

Use one color for the backgrounds. ( can be a shade of the primary eg either dark or light) its up to you.

**Review by m26unkwn — APPROVED** [review]

**Comment by PuneetP16** [comment]
> 1. Improve padding in badges
> 2. Align containers and text
> 3. Increase font-weight of badges
> 4. Change the color of badges for a better visual appeal.

Done. Thanks for the suggestions.

**Comment by PuneetP16** [comment]
> everything looks good , but please check following pointers :)
> 
> improve badges in every line.
> 
> improve color combinations in the website.
> 
> improve contrast throughout the website.
> 
> improve badge page.
> 
> Use one color for the backgrounds. ( can be a shade of the primary eg either dark or light) its up to you.

Done. Thanks for the suggestions.

File: src/components/badge/badge.css [hunk 1]
```diff
@@ -21,15 +21,17 @@
 }
 
 .color-primary-bg {
-	background-color: var(--COLOR-PRIMARY);
+	background-color: var(--COLOR-PRIMARY-DARK);
+    color:white;
```
Comment (bhtibrewal) [inline src/components/badge/badge.css:25]: Use variables to maintain consistency

**Review by bhtibrewal — APPROVED** [review]
Naming convention for classes and variables is really good

**Pull request merged**